tests/bgp/test_ipv6_bgp_scale.py

```diff
@@ -414,6 +422,64 @@ def compress_expected_routes(expected_routes):
     return b64_str
 
 
+def get_RP_start_time(duthost, connection_type, action, LOG_STAMP, syslog='/var/log/syslog'):
```

Comment:
prefer full name than short names - `get_route_programming_start_time_from_syslog`.

explicit is beauty. the function should do what it says - reversely, the function should be named to show what it does.